### DISCUSSION TITLE
Installation problem 'npm'

### DISCUSSION THREAD

**Discussion by MargaretNagawa** [thegetty/quire#231] [Get Help] [answered]
Hi!

I am encountering npm warnings as I try to install Quire on Mac. I tried running the npm audit but it didn't help. I have deleted node-modules from lib but that did not solve the problem either. I have followed answers you gave to #102 and to 1000camels to no avail. 

Please help me solve the error message below.

Thank you!

npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated axios@0.19.2: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410

changed 607 packages, and audited 608 packages in 15s

33 packages are looking for funding
  run `npm fund` for details

15 vulnerabilities (8 moderate, 7 high)

To address issues that do not require attention, run:
  npm audit fix

To address all issues possible (including breaking changes), run:
  npm audit fix --force

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.

Thank you,
Margaret

**Comment by geealbers** [comment, marked as answer]
Hi @MargaretNagawa, a lot of the output you're seeing in Terminal is not generated by Quire but by the other software being installed, and typically we suggest ignoring any `npm WARN` messages and only worry about `npm ERR!`. I suggest running `npm install --global @thegetty/quire-cli` one more time, and then test ingwith `quire --help`. If that comes back as "command not found" then there's some other error and it would be helpful in that case to see the entire output from your Terminal. Otherwise, installation should be ok. You can proceed by creating a new project and previewing it in your browser. If at that point you have any further trouble, feel free to post here again.

**Reply by MargaretNagawa** [reply]
Thank you so much, Greg! I have succeeded in downloading Quire.

Cheers!
